site/_data/demos/navigation.yml

```diff
@@ -74,6 +74,10 @@
     items:
       - title: Multiple Axes
         path: /react/chart/demos/combination/multiple-axes
+  - title: Zoom and Pan
+    items:
+      - title: Zoom and Pan
+        path: /react/chart/demos/zoom-and-pan/scatter
```

Comment:
The demo is not working on the site. The folder with name which contain hyphens is not parsing correctly

Reply:
Updated.
(though locally site was successfully build)